src/System.Net.Sockets/src/System/Net/Sockets/AcceptOverlappedAsyncResult.Windows.cs

```diff
@@ -101,8 +96,6 @@ internal void SetUnmanagedStructures(byte[] buffer, int addressBufferLength)
 
         private void LogBuffer(long size)
         {
-            if (!NetEventSource.IsEnabled) return;
```

Comment:
Yeah, that's unfortunate, because it would be nice to ensure we're never calling these kinds of methods by mistake.  I suppose we could add something like "NetEventSource.WasEverEnabled", but that's probably beyond the scope here.